homeassistant/components/plex/config_flow.py

```diff
@@ -282,6 +286,12 @@ def __init__(self, config_entry):
                         CONF_SHOW_ALL_CONTROLS,
                         default=self.options[MP_DOMAIN][CONF_SHOW_ALL_CONTROLS],
                     ): bool,
+                    vol.Required(
+                        CONF_IGNORE_SHARED_USERS,
+                        default=self.options[MP_DOMAIN].get(
+                            CONF_IGNORE_SHARED_USERS, False
```

Comment:
Should we make the default `True` ? I assume most users do not care if the Chrome browser of their shared friend is playing Plex.

Reply:
I didn't think it was obvious for everyone. I left the current default to avoid adding confusion for why some clients stop appearing.